### PR TITLE
fix log level param parsing

### PR DESCRIPTION
**Pull request by eguzki**

### CODE REVIEW

File: pkg/apis/apps/v1alpha1/apicast_types.go [hunk 1]
```diff
@@ -36,7 +36,8 @@ type APIcastSpec struct {
 	// +optional
 	ConfigurationLoadMode *int64 `json:"configurationLoadMode,omitempty"` // APICAST_CONFIGURATION_LOADER
 	// +optional
-	LogLevel *LogLevelType `json:"logLevel,omitempty"` // APICAST_LOG_LEVEL
+	// +kubebuilder:validation:Enum=debug;info;notice;warn;error;crit;alert;emerg
+	LogLevel *string `json:"logLevel,omitempty"` // APICAST_LOG_LEVEL
```
Comment (miguelsorianod) [inline pkg/apis/apps/v1alpha1/apicast_types.go]: If we decide to replace the typed string for a pure string then the following part of code in apicast_logic_reconciler.go is not needed anymore

```
	var logLevel *string
	if r.APIcastCR.Spec.LogLevel != nil {
		res := string(*r.APIcastCR.Spec.LogLevel)
		logLevel = &res
	}
```

Comment (eguzki) [reply pkg/apis/apps/v1alpha1/apicast_types.go]: :+1:

**Comment by eguzki** [comment]
ready for review @miguelsorianod

**Comment by eguzki** [comment]
"close and comment" button should have confirmation XD

**Pull request merged**